service/history/hsm/tree.go

```diff
@@ -123,10 +128,27 @@ type Node struct {
 	backend     NodeBackend
 }
 
-// NewRoot creates a new root [Node].
+type RootNode struct {
```

Comment:
Let's keep the same structure as before, you can pass the same opLog instance to all of the nodes.

Reply:
Fixed